### PR TITLE
v5.12.1

### PR DESCRIPTION
**Pull request by DanailH**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.12.1
+
+_Jun 9, 2022_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```
Comment (DanailH) [inline CHANGELOG.md:12]: As far as I see most of it is bug fixes. Is there something specific we would like to add here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +75,8 @@ We'd like to offer a big thanks to the 15 contributors who made this release pos
 
 ### `@mui/x-data-grid@v5.12.0` / `@mui/x-data-grid-pro@v5.12.0` / `@mui/x-data-grid-premium@v5.12.0`
 
+#### Changes
```
Comment (DanailH) [inline CHANGELOG.md:78]: This was missed last time. Its just for consistency

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,60 @@
...
+
+- [DataGrid] Add `reason` to `onFilterModelChange` (#4938) @m4theushw
+- [DataGrid] Restore focus after dismissing the column menu (#5027) @m4theushw
+- [DataGrid] Update quick filter input when model is modified (#5013) @alexfauquette
+- [DataGrid] Fix implicit dependency on react-dom (#5121) @oliviertassinari
+- [DataGrid] Support `getRowId` in row reordering (#5093) @flaviendelangle
+- [DataGridPro] Fix column resizing in RTL mode (#4989) @cherniavskii
+- [DataGridPro] Fix column resizing on touchscreen (#5056) @cherniavskii
+- [l10n] Update `jaJP` locale (#5122) @hikotq
+- [l10n] Update `ruRU` locale (#5069) @Artboomy
```
Comment (m4theushw) [inline CHANGELOG.md:26]:
```suggestion
- [l10n] Update Japanese (ja-JP) locale (#5122) @hikotq
- [l10n] Update Russian (ru-RU) locale (#5069) @Artboomy
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,60 @@
...
+### `@mui/x-date-pickers@5.0.0-alpha.6` / `@mui/x-date-pickers-pro@5.0.0-alpha.6`
+
+#### Changes
+
+- [pickers] Fix usage of `maxDate` / `minDate` / `disableFuture` and `disablePast` (#5081) @flaviendelangle
+- [pickers] Infer mask from `inputFormat` (#5060) @alexfauquette
+- [pickers] Manage input value without using the focus (#4486) @alexfauquette
+- [pickers] Use new localization for doc examples (#5097) @flaviendelangle
+- [pickers] Fix `shouldDisableDate` in range pickers (#5123) @flaviendelangle
+- [l10n] Add Brazilian Portuguese (`pt-BR`) locale for date pickers (#5100) @jardelnovaes
```
Comment (flaviendelangle) [inline CHANGELOG.md:37]:
```suggestion
- [l10n] Add Brazilian Portuguese (pt-BR) locale (#5100) @jardelnovaes
```

File: CHANGELOG.md (hunk 1 above)
Comment (flaviendelangle) [reply CHANGELOG.md:12]: For the pickers, the main new thing is the warning on the lab (not directly related to this release though).

Or #5060 (@alexfauquette)

File: CHANGELOG.md (hunk 2 above)
Comment (DanailH) [reply CHANGELOG.md:78]: yes, but I checked the older releases and it seems we are always adding it. It does seem more structured with it.

**Review by alexfauquette — APPROVED** [review]

**Review by flaviendelangle — APPROVED** [review]

**Pull request merged**